datahub-web-react/src/app/ingest/source/builder/RecipeForm/SecretField/SecretField.tsx

```diff
@@ -36,25 +36,30 @@ interface SecretFieldProps {
 
 function SecretField({ field, secrets, removeMargin, refetchSecrets }: SecretFieldProps) {
     return (
-        <StyledFormItem name={field.name} label={field.label} tooltip={field.tooltip} removeMargin={!!removeMargin}>
-            <Select
-                showSearch
-                filterOption={(input, option) => !!option?.children.toLowerCase().includes(input.toLowerCase())}
-                dropdownRender={(menu) => (
-                    <>
-                        {menu}
-                        <StyledDivider />
-                        <CreateSecretButton refetchSecrets={refetchSecrets} />
-                    </>
-                )}
-            >
-                {secrets.map((secret) => (
-                    <Select.Option key={secret.urn} value={`\${${secret.name}}`}>
-                        {secret.name}
-                    </Select.Option>
-                ))}
-            </Select>
-        </StyledFormItem>
+        <>
```

Comment:
why add the react fragments here? I think you can remove!

Reply:
Fixed and Commited!